meeteval/viz/visualize.py

```diff
@@ -603,6 +641,21 @@ def load_cdn(name, url):
         d3 = f'<script>{load_cdn("d3.js", cdn["d3"])}</script>'
         # font_awesome = f'<style>{load_cdn("d3font_awesome.css", cdn["font_awesome"])}</style>'
 
+        # Determine the number of minimaps based on the number of utterances. If it's a large number, 
+        # use two minimaps, else use one
+        # Typical number of utterances (`max([len(v) for v in meeteval.io.load('ref.seglst.json').groupby('session_id').values()]) * 2`):
+        #   - libricss: 250
+        #   - notsofar: 652
+        #   - dipco: 2568
+        #   - chime6: 6738
+        if self.num_minimaps == 'auto':
+            if len(self.data['utterances']) > 1000:
```

Comment:
Use length as indicator. Otherwise, lots of overlapping speech triggers the second minimap.

Reply:
Done. Set the threshold to 300s. LibriCSS gets 2 minimaps